jin/2주차 과제/20.8.py

```diff
@@ -0,0 +1,12 @@
+a, b = map(int, input().split())
+c = b+1
+
+for i in range(a,c,1):
```

Comment:
```suggestion
for i in range(a,b+1):
```
변수 C를 굳이 안 만들어 줘도 될 것 같아영 !!